test/general/interface_check.pass.cpp

```diff
@@ -0,0 +1,54 @@
+// -*- C++ -*-
+//===-- interface_check.pass.cpp --------------------------------------------===//
+//
+// Copyright (C) 2021 Intel Corporation
```

Comment:
According to the changes in https://github.com/oneapi-src/oneDPL/pull/94 year is not needed anymore in the copyright. The PR also contains the script that can be applied to all files